src/types/modules/google.d.ts

```diff
@@ -0,0 +1,36 @@
+type Response = {
+    credential: string;
+};
```

Comment:
What about exporting this type and using it on `src/components/SignInButtons/GoogleSignIn/index.website.tsx`?